subtests/docker_cli/kill_stopped/kill_utils.py

```diff
@@ -80,7 +80,7 @@ def _init_container_normal(self, name):
         subargs.append("bash")
```

Comment:
Just FYI: There's an open issue about the duplicated ``kill_utils.py``.  I haven't looked too closely, but I believe all the separate subtests which use it can/should be combined.  This will make for a massive test, but it's okay b/c it's signalling related and a part of docker that needs through testing.